packages/selfcare-client-users-updater-consumer/src/services/selfcareClientUsersUpdaterProcessor.ts

```diff
@@ -0,0 +1,118 @@
+import {
+  getInteropHeaders,
+  logger,
+  RefreshableInteropToken,
+  userRole,
+} from "pagopa-interop-commons";
+import { EachMessagePayload } from "kafkajs";
+import {
+  generateId,
+  CorrelationId,
+  genericInternalError,
+} from "pagopa-interop-models";
+import { AuthorizationProcessClient } from "../clients/authorizationProcessClient.js";
+
+// eslint-disable-next-line @typescript-eslint/explicit-function-return-type
+export function selfcareClientUsersUpdaterProcessorBuilder(
+  refreshableToken: RefreshableInteropToken,
+  authorizationProcessClient: AuthorizationProcessClient,
+  productId: string,
+  allowedOriginsUuid: string[]
```

Comment:
Usually when we adopt Uuid  we using branded string, in this case name differ from string type, I suggest to adopt the same naming convention adopted in other services o simply "allowedOrigins".
```suggestion
  selfcareUsersAllowedOrigins: producerAllowedOrigins
```